src/store/sagas/rootSaga.js

```diff
@@ -0,0 +1,16 @@
+import { all } from 'redux-saga/effects';
+
+import { watchUser, watchLoginUser, watchlogoutUser, watchUpdateUserAsync } from './userSaga';
+import { watchGetAllTasks, watchupdateAllTasks, watchdeleteCurrentTask } from './tasksSaga';
+
+export function* rootSaga() {
+  yield all([
+    watchUser(),
+    watchLoginUser(),
+    watchlogoutUser(),
+    watchUpdateUserAsync(),
+    watchGetAllTasks(),
+    watchupdateAllTasks(),
+    watchdeleteCurrentTask()
+  ]);
```

Comment:
do not use billions of watchers, rather it's better to create one

```
export function* rootSaga() {
  yield all([
    watchUser(),
    watchLoginUser(),
    watchlogoutUser(),
    watchUpdateUserAsync(),
    ...
```

---> 

```
export function* rootSaga() {
  yield all([
    watchUser(),
    ...
```

in userSaga.js:
```
export function* watchUser() {
  yield all [
     takeEvery(GET_USER_ASYNC, getUser),
     takeEvery(LOGIN_USER_ASYNC, loginUser),
     takeEvery(LOGOUT_USER_ASYNC, logoutUser),
    takeEvery(UPDATE_USER_ASYNC, update)
  ]
}
```